src/react/features/connector/mockResponses/GET_HOME.json

```diff
@@ -12,5 +12,11 @@
       "text": "My Life",
       "image": "https://upload.wikimedia.org/wikipedia/commons/9/99/The_love_of_my_life.jpg"
     }
+  },
+  "data": {
+    "focus": {
+      "room": "room1",
+      "diory": "diory1"
+    }
```

Comment:
Exactly the same file as `public/home.json`? :D 

Maybe this thing should be fixed in https://github.com/DioryMe/diory-browser-electron/issues/25 as well...

Reply:
Github is not working well... I already answered but no answer here... 
no, this is for testing. Just happens to be same now but 25 will change that.

Reply:
👍